src/JK/CommonBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,3 @@
+services:
+  jk.common.controller_abstract:
+       class: JK\CommonBundle\Controller\AbstractController
```

Comment:
Please move FQCN to parameters, like

testlab.common.controller.abstract.class: TestLab\CommonBundle\Controller\AbstractController

class: %testlab.common.controller.abstract.class%